### PR TITLE
Make ActiveElement endpoint W3C compatible

### PR DESCRIPTION
**Pull request by andrewnicols**
As defined in the specification 12.2.6:

HTTP Method   URI Template
GET           /session/{session id}/element/active

Source: https://www.w3.org/TR/webdriver/#get-active-element

### CODE REVIEW

File: tests/functional/RemoteTargetLocatorTest.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{
+    /**
+     * @cover ::activeElement
+     */
+    public function testActiveElement()
+    {
+        $this->driver->get($this->getTestPageUrl('index.html'));
+
+        $activeElement = $this->driver->switchTo()->activeElement();
+        $this->assertInstanceOf(\Facebook\WebDriver\Remote\RemoteWebElement::class, $activeElement);
```
Comment (OndraM) [inline tests/functional/RemoteTargetLocatorTest.php]: You can import the class and have there only:

```suggestion
        $this->assertInstanceOf(RemoteWebElement::class, $activeElement);
```

File: tests/functional/RemoteTargetLocatorTest.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        $this->assertEquals('body', $activeElement->getTagName());
```
Comment (OndraM) [inline tests/functional/RemoteTargetLocatorTest.php]:
```suggestion
        $this->assertSame('body', $activeElement->getTagName());
```

File: tests/functional/RemoteTargetLocatorTest.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+        $this->driver->findElement(WebDriverBy::name('test_name'))->click();
+        $activeElement = $this->driver->switchTo()->activeElement();
+        $this->assertEquals('input', $activeElement->getTagName());
```
Comment (OndraM) [inline tests/functional/RemoteTargetLocatorTest.php]:
```suggestion
        $this->assertSame('input', $activeElement->getTagName());
```

File: tests/functional/RemoteTargetLocatorTest.php [hunk 4]
```diff
@@ -52,4 +52,21 @@ public function testShouldSwitchToWindow()
...
+        $this->assertEquals('test_name', $activeElement->getAttribute('name'));
```
Comment (OndraM) [inline tests/functional/RemoteTargetLocatorTest.php]:
```suggestion
        $this->assertSame('test_name', $activeElement->getAttribute('name'));
```

**Comment by andrewnicols** [comment]
Rebased on latest community branch.
This issue blocks #681.

**Comment by andrewnicols** [comment]
I think I just missed your comment. I'll sort those now.

**Comment by OndraM** [comment]
@andrewnicols No problem, it seems we worked on in at the same time :smile: , thanks!

**Comment by OndraM** [comment]
(Oh, wrong button pressed, sorry 😬)

**Comment by andrewnicols** [comment]
I've swapped those assertEquals to assertSame.
Originally I had that test in a namespace of `Facebook\WebDriver\Remote` so it didn't need importing (though I don't know why I specified the whole thing anyway).

Should all be ready to go now. Confirmed unit tests still pass on the usual browser/driver combinations.

**Pull request merged**